Templates/SaaSFileTrigger-Batch/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "SaaSFileTrigger - Batch",
+    "description": "A Batch function that will be run whenever a file is added to a SaaS provider.",
```

Comment:
What if the file is added when the function isn't running, or what if the trigger fails (e.g. due to some non user issue) before the user function is called. Is that addition lost, or does ApiHub manage determining whether a notification is actually processed successfully (i.e. retry semantics)?